data/genomics/sarscov2/genome/baits.interval_list

```diff
@@ -0,0 +1,7 @@
+@HD	VN:1.6	SO:coordinate
```

Comment:
I wonder whether we should put these files in a `picard` folder instead?

Reply:
done!